pkg/daemon/daemon.go

```diff
@@ -2017,3 +2032,27 @@ func forceFileExists() bool {
 
 	return false
 }
+
+func (dn *Daemon) createCAPIBootstrapSentinelFile() error {
+	// No need to create the file on control plane machines
+	if dn.isControlPlane {
+		return nil
+	}
```

Comment:
Why is this? Since there's no harm in it, should we just always create it? That would minimise the diff between control plane and workers

Reply:
Yep, I removed this condition